Doc/library/site.rst

```diff
@@ -57,6 +57,19 @@ directory rather than a file.  No item is added to ``sys.path`` more than
 once.  Blank lines and lines beginning with ``#`` are skipped.  Lines starting
 with ``import`` (followed by space or tab) are executed.
 
+.. note::
+
+   An executable line in a :file:`.pth` file is run at every Python startup,
+   regardless of whether a particular module is actually going to be used.
+   Its impact should thus be kept to a minimum.
+   The primary intended purpose of the lines is to make the
```

Comment:
```suggestion
   The primary intended purpose of import lines is to make the
```

Reply:
I've changed that to "executable lines": "import line" is a bit beside their point. This also makes for consistent terminology: "executable line" is now the official term for the construct.